### PR TITLE
adding css to student showcase

### PR DESCRIPTION
**Pull request by AshleaSwindell**

### CODE REVIEW

File: theme/style.css [hunk 1]
```diff
@@ -16,3 +16,28 @@
...
+
+.list {
+  list-style: none;
+}
+
+.list a:hover {
+  background-color: yellow;
+}
+
+ul li a {
```
Comment (NAlexPear) [inline theme/style.css]: This rule is probably a bit too broad... there are going to be a lot of anchor tags in list items on this page, and not all of them should be green 😬

**Comment by AshleaSwindell** [comment]
I added a class to the unordered list, does this narrow it down enough?

File: index.html [hunk 2]
```diff
@@ -58,16 +58,18 @@ <h3>Contacts: <a href="jakepelech@gmail.com">E-Mail</a> <a href="https://github.
             <p>"Time is an illusion.  Lunchtime doubly so." ~Douglas Adams</p>
             <p>"Bowties and Hitchhiker quotes are for nerrrrds." ~Everyone</p>
         </div>
-        <div>
-          <img src="https://avatars0.githubusercontent.com/u/38234207?s=400&u=ff4f764c6655251d3f10e610b76d2141a584d989&v=4" alt="My photo">
+        <div id="ashlea">
+          <img src="https://avatars0.githubusercontent.com/u/38234207?s=400&u=ff4f764c6655251d3f10e610b76d2141a584d989&v=4" alt="My photo" width=300px>
```
Comment (NAlexPear) [inline index.html]: no inline styles, please!

Comment (AshleaSwindell) [reply index.html]: okay it should be good to go!

**Pull request merged**